html/base/inputs/HugeRadio.stories.js

```diff
@@ -0,0 +1,921 @@
+  useEffect(() => {
+    radioInput();
+  }, []);
+
+  return `
+    <div class="sprk-b-InputContainer">
+      <fieldset class="sprk-b-Fieldset">
+        <legend class="sprk-b-Legend">
+          <label class="sprk-b-Label">
+            Radio Group Label
+          </label>
+        </legend>
+
+        <div
+          class="sprk-b-SelectionContainer sprk-b-Radio sprk-b-Radio--huge"
+          data-sprk-input="radio"
+        >
+          <input
+            class="sprk-b-Radio__input"
+            id="radio-1"
+            data-id="radio-1"
+            type="radio"
+            name="radio"
+            aria-describedby="radio--error-container"
+          >
+
+          <label
+            for="radio-1"
+            class="sprk-b-Label sprk-b-Label--inline sprk-b-Radio__label"
+          >
+            Radio Item 1
+          </label>
+        </div>
+
+        <div
+          class="sprk-b-SelectionContainer sprk-b-Radio sprk-b-Radio--huge"
+          data-sprk-input="radio"
+        >
+          <input
+            class="sprk-b-Radio__input"
+            id="radio-2"
+            data-id="radio-2"
+            type="radio"
+            name="radio"
+            aria-describedby="radio--error-container"
+          >
+
+          <label
+            for="radio-2"
+            class="sprk-b-Label sprk-b-Label--inline sprk-b-Radio__label"
+          >
+            Radio Item 2
+          </label>
+        </div>
+
+        <div
+          class="sprk-b-SelectionContainer sprk-b-Radio sprk-b-Radio--huge"
+          data-sprk-input="radio"
+        >
+          <input
+            class="sprk-b-Radio__input"
+            id="radio-3"
+            data-id="radio-3"
+            type="radio"
+            name="radio"
+            aria-describedby="radio--error-container"
+          >
+          <label
+            for="radio-3"
+            class="sprk-b-Label sprk-b-Label--inline sprk-b-Radio__label"
+          >
+            Radio Item 3
+          </label>
+        </div>
+      </fieldset>
+
+      <div
+        class="sprk-b-ErrorContainer"
+        id="radio--error-container"
+      ></div>
+    </div>
+  `
+};
+
+defaultStory.story = {
+  name: 'Default',
+  parameters: {
+    jest: ['radioInput'],
+    docs: { iframeHeight: 300 },
+  }
+};
+
+export const layoutTwo = () => {
+  useEffect(() => {
+    radioInput();
+  }, []);
+
+  return `
+    <div class="sprk-b-InputContainer sprk-b-InputContainer--full">
+      <fieldset class="sprk-b-Fieldset">
+        <legend class="sprk-b-Legend">
+          <label class="sprk-b-Label">
+            Radio Group Label
+          </label>
+        </legend>
+
+        <div class="sprk-o-Stack sprk-o-Stack--split@s">
+          <div class="sprk-o-Stack__item sprk-o-Stack__item--flex@s">        
+            <div
+              class="
+                sprk-b-SelectionContainer
+                sprk-b-Radio
+                sprk-b-Radio--huge
+                sprk-u-phs
```

Comment:
Lets use the built in spacing settings from stack instead of this utility (sprk-u-phs).

Reply:
@afebbraro  This is what we originally tried! Here's what we found (example using --tiny stack spacing)
![image](https://user-images.githubusercontent.com/4342363/83179570-329d8280-a0f0-11ea-8410-caf09e598eda.png)
But then it stacks and doubles on top of each other
![image](https://user-images.githubusercontent.com/4342363/83179603-3a5d2700-a0f0-11ea-8bff-7a7c215c23ee.png)

Reply:
Essentially, we don't want to have space added to the top and bottom of the individual checkboxes and there isn't an option in stack to just add spacing in between the items.

Reply:
stack only adds margin bottom